rest/blip_client_test.go

```diff
@@ -117,6 +135,15 @@ func (btr *BlipTesterReplicator) initHandlers(btc *BlipTesterClient) {
 	btr.bt.blipContext.HandlerForProfile[db.MessageChanges] = func(msg *blip.Message) {
 		btr.storeMessage(msg)
 
+		collection := msg.Properties[db.BlipCollection]
+
+		// FIXME: Remove when we have CBG-2264 implemented
+		if len(btc.Collections) > 0 {
+			collection = btc.Collections[0]
+		}
+
+		btcr := btc.CollectionClients[collection]
```

Comment:
This doesn't seem right - [per spec](https://docs.google.com/document/d/1kbPUuWgtnAgV9Oj55_soWmIPA8f9v3RBrjMXVvbn4p4/edit#heading=h.6zpikbtvy0cn), the `collection` property is the index into the original `getCollections` body array, but it looks like `btc.CollectionClients` is indexed by the collection name. Am I missing something?

Reply:
+1 - something looks off here about how we're accessing vs. how `initCollectionReplication` is adding items to the map

Reply:
Yeah fair enough, missed that, talked to Marks about this, this morning. Going to keep it internally indexed by name though and convert it. I think this is easier for debugging (as its a test client) and think it makes it easier to follow.